docs/content/doc/usage/fail2ban-setup.en-us.md

```diff
@@ -108,3 +108,12 @@ this to your Nginx configuration so that IPs don't show up as 127.0.0.1:
 ```
 proxy_set_header X-Real-IP $remote_addr;
 ```
+
+The security options in `app.ini` need to be adjusted to allow the interpretation of the headers
+as well as the list of IP addresses and networks that describe trusted proxy servers
+(find more information in the corresponding [PR](https://github.com/go-gitea/gitea/pull/14959)).
```

Comment:
It would be better to add link to docs somewhere instead of PR

Reply:
```suggestion
(See the [configuration cheat sheet](https://docs.gitea.io/en-us/config-cheat-sheet/#security-security) for more information).
```